packages/compartment-mapper/src/assemble.js

```diff
@@ -82,6 +104,11 @@ const makeModuleMapHook = (
 
       if (foreignModuleSpecifier !== undefined) {
         const { compartment: foreignCompartmentName } = scopeDescriptor;
+        if (foreignCompartmentName === undefined) {
+          throw new Error(
```

Comment:
Use assert?

Reply:
I’d like to avoid SES-shim coupling for now. This should work with other Compartment implementations some day and should be able to work without lockdown, to prove out Compartment as a generally suitable module loader.

Reply:
ah. right... that sort of occurred to me after I made the comment... `@agoric/assert` isn't handy here.